test/kubemark/resources/manifests/addons/kubemark-rbac-bindings/kubelet-binding.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1alpha1
+kind: ClusterRoleBinding
+metadata:
+  name: kubelet-cluster-admin
+  labels:
+    kubernetes.io/cluster-service: "true"
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
This one surprises me a bit.  Why does the kubelet need to be a cluster-admin?  If our standard kubelet role is wrong, we need to fix it.